core/extensions.js

```diff
@@ -176,6 +176,41 @@ Blockly.Extensions.checkDropdownOptionsInTable_ =
   };
 
 /**
+ * Builds an extension function that will install a dynamic tooltip. The
+ * tooltip message should include a the string '%1' and that string will be
```

Comment:
`Blockly.Msg.MATH_CHANGE_TOOLTIP`: https://github.com/google/blockly/blob/develop/msg/messages.js#L470
`Blockly.Msg.CONTROLS_FOR_TOOLTIP`: https://github.com/google/blockly/blob/develop/msg/messages.js#L199
`Blockly.Msg.CONTROLS_FOREACH_TOOLTIP`: https://github.com/google/blockly/blob/develop/msg/messages.js#L217

Make sure you are not referencing these them prior to when the messages.js (or appropriate translation) is loaded.  For instance, `Blockly.Extensions.buildTooltipForDropdown(..)` uses `Blockly.utils.runAfterPageLoad(..)` to defer dereferencing the tooltips strings.